lighthouse-core/test/fraggle-rock/gather/mock-driver.js

```diff
@@ -218,33 +220,33 @@ function createMockContext() {
 }
 
 function mockDriverSubmodules() {
-  const navigationMock = {gotoURL: jest.fn()};
+  const navigationMock = {gotoURL: fnAny()};
   const prepareMock = {
-    prepareThrottlingAndNetwork: jest.fn(),
-    prepareTargetForTimespanMode: jest.fn(),
-    prepareTargetForNavigationMode: jest.fn(),
-    prepareTargetForIndividualNavigation: jest.fn(),
+    prepareThrottlingAndNetwork: fnAny(),
+    prepareTargetForTimespanMode: fnAny(),
+    prepareTargetForNavigationMode: fnAny(),
+    prepareTargetForIndividualNavigation: fnAny(),
   };
-  const storageMock = {clearDataForOrigin: jest.fn()};
+  const storageMock = {clearDataForOrigin: fnAny()};
   const emulationMock = {
-    clearThrottling: jest.fn(),
-    emulate: jest.fn(),
+    clearThrottling: fnAny(),
+    emulate: fnAny(),
   };
   const networkMock = {
-    fetchResponseBodyFromCache: jest.fn(),
+    fetchResponseBodyFromCache: fnAny(),
   };
   const targetManagerMock = mockTargetManagerModule();
 
   function reset() {
-    navigationMock.gotoURL = jest.fn().mockResolvedValue({finalUrl: 'https://example.com', warnings: [], timedOut: false});
-    prepareMock.prepareThrottlingAndNetwork = jest.fn().mockResolvedValue(undefined);
-    prepareMock.prepareTargetForTimespanMode = jest.fn().mockResolvedValue(undefined);
-    prepareMock.prepareTargetForNavigationMode = jest.fn().mockResolvedValue({warnings: []});
-    prepareMock.prepareTargetForIndividualNavigation = jest.fn().mockResolvedValue({warnings: []});
-    storageMock.clearDataForOrigin = jest.fn();
-    emulationMock.clearThrottling = jest.fn();
-    emulationMock.emulate = jest.fn();
-    networkMock.fetchResponseBodyFromCache = jest.fn().mockResolvedValue('');
+    navigationMock.gotoURL = fnAny().mockResolvedValue({finalUrl: 'https://example.com', warnings: [], timedOut: false});
```

Comment:
the `jest` here is now using types privided from jest directly via `@jest/globals`, whereas before it was from `@types/jest` (which is pinned to an older set of types). In the newer version, jest uses `unknown` where we relied on `any` for our types to work.

It's pretty awkward to use jsdoc to apply the correct template arguments, so I introduced a `fnAny` helper.